roles/network_plugin/cilium/defaults/main.yml

```diff
@@ -348,3 +348,6 @@ cilium_certgen_args:
 cilium_clusterrole_rules_operator_extra_vars: []
 cilium_enable_host_firewall: false
 cilium_policy_audit_mode: false
+
+# cilium_hubble_export_file_max_backups: "5"
+# cilium_hubble_export_file_max_size_mb: "10"
```

Comment:
No, I meant here, sorry.
Those should not be commented out (assuming they are the correct upstream default values)